src/main/java/com/github/princesslana/eriscasper/data/Data.java

```diff
@@ -35,4 +38,11 @@ public static String toJsonString(Object obj) throws DataException {
       throw new DataException(e);
     }
   }
+
+  public static String toQueryString(Object obj) {
+    Iterable<Map.Entry<String, JsonNode>> iterableFields = () -> jackson.valueToTree(obj).fields();
+    return StreamSupport.stream(iterableFields.spliterator(), false)
+        .map(e -> e.getKey() + "=" + e.getValue())
+        .collect(Collectors.joining("&"));
```

Comment:
Didn't end up too bad in the end 👍

Reply:
Yeah I think it looks good and shows exactly what it's doing fairly well, not too heavy using exactly what it needs. Not sure of a more efficient way of doing it.